resources/language/resource.language.fr_fr/strings.po

```diff
@@ -764,47 +764,48 @@ msgstr "le film"
 
 msgctxt "#30190"
 msgid "TV show"
-msgstr "la série télé"
+msgstr "la série"
 
 msgctxt "#30191"
 msgid "Do you want to export NFO files for the {} added to 'My List'?"
 msgstr "Voulez-vous exporter les fichiers NFO pour {} ajouté à 'Ma Liste' ?"
 
 msgctxt "#30192"
 msgid "\r\nThese files are used by the provider information scrapers to integrate movies and tv shows info, useful with bonus episodes or director's cut"
-msgstr "\r\nCes fichiers sont utilisés par le fournisseur d'information pour intégrer les infos des films et des séries télé. Utile pour des épisodes bonus ou des découpages spéciaux"
+msgstr "\r\nCes fichiers sont utilisés par les fournisseurs d'informations pour intégrer les infos des films et des séries, ils sont utiles pour les épisodes bonus ou les versions intégrales"
+
 
 msgctxt "#30193"
 msgid "Include all information in NFO files (use only with 'Local Information' scraper)"
-msgstr "Inclure toutes les informations dans les fichiers NFO (utiliser uniquement avec le fournisseur d'information 'Local Information only')"
+msgstr "Inclure toutes les informations dans les fichiers NFO (à utiliser uniquement avec le fournisseur d'informations 'Local information only')"
```

Comment:
Fix: 'Local information' without "only" term

Reply:
Ok, will fix it in my next commit.